src/main/java/com/example/bookshop/exception/CustomGlobalExceptionHandler.java

```diff
@@ -44,6 +44,15 @@ public ResponseEntity<Object> handleEntityNotFoundException(EntityNotFoundExcept
         return new ResponseEntity<>(body, HttpStatus.NOT_FOUND);
     }
 
+    @ExceptionHandler(RegistrationException.class)
+    public ResponseEntity<Object> handleRegistrationException(RegistrationException ex) {
+        Map<String, Object> body = new LinkedHashMap<>();
+        body.put("timestamp", LocalDateTime.now());
+        body.put("status", HttpStatus.BAD_REQUEST);
+        body.put("message", ex.getMessage());
+        return new ResponseEntity<>(body, HttpStatus.BAD_REQUEST);
```

Comment:
This code is pretty equal to handleEntityNotFoundException method body. 
Please, avoid code duplication.